software/qst.yml

```
@@ -0,0 +1,12 @@
+name: "QST"
+website_url: "https://qstonline.org"
+source_code_url: "https://sourceforge.net/projects/qstonline/"
+description: "QST, the worlds unparalleled open source online assessment software. From a quick quiz on your phone to large scale, high stakes, proctored desktop testing, we make it easy, secure and economical."
```

```suggestion
description: "Online assessment software. From a quick quiz on your phone to large scale, high stakes, proctored desktop testing, easy, secure and economical."
```

Simple/more neutral description.

---

@ryangurn ping